### PR TITLE
Esha/profile style

### PR DESCRIPTION
**Pull request by eshabansiya**
### 🦋 Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

Frontend: Followed [Figma](https://www.figma.com/design/ONiX8NmbtginHjmnZjTonb/ECOVET-GLOBAL?node-id=617-1224&p=f&m=dev)
 - Added black70 into Colors 
 - Based on Dylan's Reset Password styling

Sessions: 
Query: createSession(templateId, userGroupId)
- creates a new session with a unique uuid
- assigns the template_id, user_group_id, and sets async to False
- template_id is hard-coded as well 
- _returns the new session_id_ 
Query: assignSession(userId, sessionId)
- updates a user's session_id
Role Selection page:
- Next button to make a new session (createSession) and then to add the session to self profile and the participants selected (assignSessions)
- Everytime next is clicked it will make a new session (dont click it too many times)
- updates a message displayed
- Tested with: const userGroupId = '0b73ed2d-61c3-472e-b361-edaa88f27622';

### 🦧 Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

Assigning Sessions:
Screenshot 2025-11-09 at 3 57 29 PM

Onboarding Frontend:
Screenshot 2025-11-09 at 3 55 05 PM

## 🐸 How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

- Test queries using a profile with a user_group_id
- Check front-end onboarding page

CC: @me-liu

### CODE REVIEW

File: styles/colors.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ const COLORS = {
   black: "#000000",
   black20: "#C7C6C3",
   black100: "#0f0f0f",
+  black70: "#4B4A49",
```
Comment (me-liu) [inline styles/colors.ts]: nit but can we move this around so its 20, 70, 100 
l o l

File: api/supabase/queries/sessions.ts [hunk 2]
```diff
@@ -39,3 +39,32 @@ export async function assignRole(userId: string, roleId: string) {
...
+
+export async function assignSession(userId: string, sessionId: string) {
+  const { data, error } = await supabase
+    .from("profile")
+    .update({ session_id: sessionId })
+    .eq("id", userId);
+  if (error) {
+    throw error;
+  }
+  return data;
```
Comment (me-liu) [inline api/supabase/queries/sessions.ts]: i believe the data will be null here so we can just remove the return statement

**Review by me-liu — COMMENTED** [review]
looks very solid!!

**Review by me-liu — APPROVED** [review]
